Remove support and tests for PowerShell and Xonsh (not available on Alpine).

--- a/man/man1/zoxide-init.1
+++ b/man/man1/zoxide-init.1
@@ -39,24 +39,6 @@
 .sp
 Note: zoxide only supports Nushell v0.37.0 and above.
 .TP
-.B powershell
-Add this to your configuration (find it by running \fBecho $profile\fR in
-PowerShell):
-.sp
-.nf
-    \fBInvoke-Expression (& {
-        $hook = if ($PSVersionTable.PSVersion.Major -lt 6) { 'prompt' } else { 'pwd' }
-        (zoxide init --hook $hook powershell | Out-String)
-    })\fR
-.fi
-.TP
-.B xonsh
-Add this to your configuration (usually \fB~/.xonshrc\fR):
-.sp
-.nf
-    \fBexecx($(zoxide init xonsh), 'exec', __xonsh__.ctx, filename='zoxide')\fR
-.fi
-.TP
 .B zsh
 Add this to your configuration (usually \fB~/.zshrc\fR):
 .sp
--- a/src/cmd/_cmd.rs
+++ b/src/cmd/_cmd.rs
@@ -89,8 +89,6 @@
     Fish,
     Nushell,
     Posix,
-    Powershell,
-    Xonsh,
     Zsh,
 }
 
--- a/src/cmd/init.rs
+++ b/src/cmd/init.rs
@@ -23,8 +23,6 @@
             InitShell::Fish => shell::Fish(opts).render(),
             InitShell::Nushell => shell::Nushell(opts).render(),
             InitShell::Posix => shell::Posix(opts).render(),
-            InitShell::Powershell => shell::Powershell(opts).render(),
-            InitShell::Xonsh => shell::Xonsh(opts).render(),
             InitShell::Zsh => shell::Zsh(opts).render(),
         }
         .context("could not render template")?;
--- a/src/shell.rs
+++ b/src/shell.rs
@@ -28,8 +28,6 @@
 make_template!(Fish, "fish.txt");
 make_template!(Nushell, "nushell.txt");
 make_template!(Posix, "posix.txt");
-make_template!(Powershell, "powershell.txt");
-make_template!(Xonsh, "xonsh.txt");
 make_template!(Zsh, "zsh.txt");
 
 #[cfg(feature = "nix-dev")]
@@ -215,6 +213,7 @@
     }
 
     #[apply(opts)]
+    #[ignore]
     fn powershell_pwsh(cmd: Option<&str>, hook: InitHook, echo: bool, resolve_symlinks: bool) {
         let opts = Opts { cmd, hook, echo, resolve_symlinks };
         let mut source = "Set-StrictMode -Version latest\n".to_string();
@@ -229,6 +228,7 @@
     }
 
     #[apply(opts)]
+    #[ignore]
     fn xonsh_black(cmd: Option<&str>, hook: InitHook, echo: bool, resolve_symlinks: bool) {
         let opts = Opts { cmd, hook, echo, resolve_symlinks };
         let mut source = Xonsh(&opts).render().unwrap();
@@ -238,6 +238,7 @@
     }
 
     #[apply(opts)]
+    #[ignore]
     fn xonsh_mypy(cmd: Option<&str>, hook: InitHook, echo: bool, resolve_symlinks: bool) {
         let opts = Opts { cmd, hook, echo, resolve_symlinks };
         let source = Xonsh(&opts).render().unwrap();
@@ -246,6 +247,7 @@
     }
 
     #[apply(opts)]
+    #[ignore]
     fn xonsh_pylint(cmd: Option<&str>, hook: InitHook, echo: bool, resolve_symlinks: bool) {
         let opts = Opts { cmd, hook, echo, resolve_symlinks };
         let mut source = Xonsh(&opts).render().unwrap();
@@ -260,6 +262,7 @@
     }
 
     #[apply(opts)]
+    #[ignore]
     fn xonsh_xonsh(cmd: Option<&str>, hook: InitHook, echo: bool, resolve_symlinks: bool) {
         let opts = Opts { cmd, hook, echo, resolve_symlinks };
         let source = Xonsh(&opts).render().unwrap();
